browser/src/map/handler/Map.Keyboard.js

```diff
@@ -680,6 +680,24 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// Cmd + Ctrl + Space on a Mac/iPad, open system emoji picker
+		if (e.keyCode === this.keyCodes.SPACE
```

Comment:
can we use new approach for it: https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.KeyboardShortcuts.ts